cmd/brew-build-bottle-pr.rb

```diff
@@ -121,7 +121,7 @@ def build_bottle(formula)
     return ohai "#{formula}: Skipping because a bottle is not needed" if formula.bottle_unneeded?
     return ohai "#{formula}: Skipping because bottles are disabled" if formula.bottle_disabled?
     return ohai "#{formula}: Skipping because it has a bottle already" if formula.bottle_specification.tag?(tag)
-    return ohai "#{formula}: Skipping because #{tap} does not support Linux" if slug(tap)[/^Homebrew/] && tap.repo != "science"
+    return ohai "#{formula}: Skipping because #{tap} does not support Linux" if slug(tap)[/^Homebrew/] && tap.repo != "science" && !slug(tap).sub(/^Homebrew\//, '').match?(HOMEBREW_OFFICIAL_REPO_PREFIXES_REGEX)
```

Comment:
```suggestion
    return ohai "#{formula}: Skipping because #{tap} does not support Linux" if tap.full_name = "Homebrew/homebrew-core"
```
We can simplify now that the Homebrew organization only has one tap.
`CoreTap.instance.full_name` returns `Homebrew/linuxbrew-core` on Linux. We can delete the unused `slug` function.